### PR TITLE
tests: test restore db

### PR DESCRIPTION
**Pull request by overvenus**
Test restore db.

### CODE REVIEW

File: tests/br_db/run.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# TODO: use backup db, once it is supported.
+# backup full
+echo "backup start..."
+run_br --pd $PD_ADDR backup full -s "local://$TEST_DIR/$DB" --ratelimit 5 --concurrency 4 --fastchecksum true
+
+run_sql "DROP DATABASE $DB;"
+
+# restore db
+echo "restore start..."
+run_br restore db --db $DB -s "local://$TEST_DIR/$DB" --pd $PD_ADDR
```
Comment (kennytm) [inline tests/br_db/run.sh]: it should check whether the restored data is correct?

Comment (overvenus) [reply tests/br_db/run.sh]: Checksum will check data integrity before finishing restore.

Comment (kennytm) [reply tests/br_db/run.sh]: Okay.

File: tests/br_key_locked/workload [hunk 2]
```diff
@@ -1,4 +1,4 @@
-recordcount=10000
+recordcount=1000
```
Comment (kennytm) [inline tests/br_key_locked/workload]: Is this change intended?

Comment (overvenus) [reply tests/br_key_locked/workload]: It speeds up br_key_locked test, resolving lock is very slow.
